config/default/manager_auth_proxy_patch.yaml

```diff
@@ -24,3 +24,4 @@ spec:
         - "--health-probe-bind-address=:8081"
         - "--metrics-bind-address=127.0.0.1:8080"
         - "--leader-elect"
+        - "--multicluster-console-port=9001"
```

Comment:
I tried this to move managed.yaml, I added --multicluster-console-port=9001 under args but it is giving an error.  it have error like invalid flag.